src/components/common/roleDropdownItems/index.js

```diff
@@ -0,0 +1,19 @@
+import { useContext } from "react";
+import { PagedataContext } from "contexts/pagedataContext";
+import { MenuItem } from "@mui/material";
+
+//Not in use (In progress)
+const RoleDropDownItems = () => {
+  const pageData = useContext(PagedataContext);
+  return pageData.roles

```

Comment:
More simpler and prettier would be to have additional `if` before `return` and in case `!pageData.roles` then return nothing. So instead of `return pageData.roles ? [...] : ""` you could do

> if (!pageData.roles) return null
>
> return pageData.roles. map([...])

Therefore less nesting and easier to read code

Reply:
I will do it . Thank you !